packages/homepage/content/docs/2-live.md

```diff
@@ -1,10 +1,12 @@
 ---
-title: Collaboration with Live
+title: v3.6
```

Comment:
You changed a docs page, so not sure if this was intended...? 🤔

Reply:
Whoops your right! This should not have been changed.